proposal/pachyderm.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Pachyderm 
+---
+
+[Pachyderm](https://www.pachyderm.com/products/) is the data foundation for machine learning and offers three different products to fit all machine learning operationalization (MLOps) needs.
+
+The Pachyderm platform brings together [version control for data](https://docs.pachyderm.com/2.0.x/concepts/data-concepts/) with the tools to build scalable [end-to-end ML/AI pipelines](https://docs.pachyderm.com/2.0.x/concepts/pipeline-concepts/) while empowering users to develop their code in any language, framework, or tool of their choice. Pachyderm has been proven to be the ideal foundation for teams looking to use ML and AI to solve real-world problems in a reliable way.
+
+*The Pachyderm platform includes the following main components*
+
+- Pachyderm File System (PFS)
+- Pachyderm pipelines
+
+*The Pachyderm Features*
+- Automated Data Versioning
+- Data-Driven Pipelines
+- Immutable Data Lineage
+
+
+## Architecture
+![Pachyderm Architecture](https://docs.pachyderm.com/2.0.x/deploy-manage/images/arch_diagram_high_level.svg)
+## User Use Cases
+- [Use Cases](https://www.pachyderm.com/use-cases/)
+- [Case Study](https://www.pachyderm.com/case-studies/)
+
+## Implementation
+
+
+![Pachyderm in Open Data Hub Architecture](../images/pachyderm_odh_architecture.png)
+
```

Comment:
Can you please add some text explaining this integration?

Reply:
I updated

Reply:
Yes, ODH will install the Pachyderm operator and will create a Pachyderm CR. I could not find an official description of Pachd. As far as I know, this is the main component for pachyderm cluster. I explained this in the README  from odh-manifests.  This is a proposal doc so I think explaining this in the odh-manifests would make more sense to me. Do you think I should copy that from odh-manifests and paste it here too?

Reply:
I briefly explained it here " To explain a little more in detail, ODH installs Pachyderm Operator and creates Pachyderm CR to deploy Pachyderm cluster."